src/ButtonIcon/index.js

```diff
@@ -3,18 +3,27 @@ import {default as PT} from 'prop-types'
 import { getElementType } from '../utils'
 import cx from 'classnames'
 
-const ButtonIcon = ({ as, extraClass, children, ...props }) => {
+const ButtonIcon = ({ as, extraClass, children, variant, ...props }) => {
   const ElementType = getElementType(ButtonIcon, { as: as, ...props })
-  return (<ElementType {...props} className={cx('ola_buttonIcon', extraClass)}>{children}</ElementType>)
+  delete props['as']
+  const styles = cx(
+    'ola_buttonIcon',
+    {[`is-${variant}`]: variant },
+    extraClass
+  )
+  return (<ElementType {...props} className={styles}>{children}</ElementType>)
 }
 
 ButtonIcon.defaultProps = {
-  as: 'button'
+  as: 'button',
+  variant: null
```

Comment:
variant cant be null in this case. PropTypes force it to be one of ['normal' or 'desctructive'] by default.

Reply:
I just replicate how variants are in Button (that is null by default). But I can change to `normal` by default.

Reply:
We must fix the null variants if they do not have a prop type null ...